playbooks/common/openshift-etcd/migrate.yml

```diff
@@ -126,7 +126,70 @@
     set_fact:
       r_etcd_migrate_success: true
 
-- include: ./scaleup.yml
+- name: Gather facts for migrated nodes
+  hosts: oo_etcd_to_config:oo_new_etcd_to_config
+  roles:
+  - openshift_etcd_facts
+
+- name: Re-configure etcd and bring the cluster up
```

Comment:
This looks okay to me.  Please update with test results.

Reply:
Had to add a few more vars, now migrates 3.7 cluster correctly